task/fbc-validation/0.1/README.md

```diff
@@ -1,12 +1,30 @@
 # fbc-validation task
 
 ## Description:
-The fbc-validation task will ensure FBC (File based catalog) components uniquely linted to ensure they're properly
-constructed as part of the build pipeline. To validate the image in build pipeline, Skopeo is used to extract
+The fbc-validation task will ensure FBC (File based catalog) components are uniquely linted to ensure they're properly
```

Comment:
README modifications will be overwritten once someone auto-generates the files using the script at https://github.com/konflux-ci/build-definitions/blob/main/hack/generate-readme.sh

Reply:
Ah.  Is there a good place to capture this info, or would I be better off omitting it entirely here?

Reply:
We can create another file in the task directory. If we want it in the README then we can modify the `generate-readme.sh` to generate the README from multiple files.

The description line that I highlighted _should_ be auto-generated from the description in the task (once you update that to the desired text). The remaining content would need to be pulled in from elsewhere.

Reply:
Got it.  Thanks, I was unfamiliar with the generation.  I'll see if I can re-flow into the task description, though I'll have to substantially change the formatting. 😄

Reply:
@arewm 
Looking back on this, and I was sure I'd seen other, manual updates to the README.  So I went back and found https://github.com/konflux-ci/build-definitions/pull/541 for e.g. 

Do you prefer that I attempt to integrate all the contributions to the task description?  It seems that the intent isn't to re-generate the README.md.

Reply:
We discussed on the architecture call today. I think that we should leave the README as auto-generated and then create additional USAGE.md and/or TROUBLESHOOTING.md files as needed for those use cases. 

Long term, we can look to merge the other files into README.md with the script for completeness.